planner-agent/src/workflows/vibeTodoCreator/phases.py

```
@@ -0,0 +1,119 @@
+            available_tools=[
+                "generate_issues",
+                "approve_issues",
+                "list_files",
+                "read_file",
+            ],
+            conversation_id=conversation_id,
+            name="Issue Validation",
+        )
+
+
+class TaskDecompositionPhase(WorkflowPhase):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)